Pipfile

```diff
@@ -16,6 +16,7 @@ semantic_version = ">=2.6.0,<3.0"
 markdown = ">=3.1.1,<4"
 bleach = ">=3.1.0,<4"
 bleach-whitelist = ">=0.0.10,<1"
+ansible-lint = ">=4.1.0,<5.0"
```

Comment:
If galaxy-importer describes dependencies in the `setup.py` file, `Pipfile` is probably not needed.
Otherwise this PR should include updated lock file as well.

Reply:
Was waiting for previous pr to merge, this is now in `setup.cfg`